src/daemon/start_mon.sh

```diff
@@ -33,8 +33,10 @@ function get_ip {
         # We don't want local scope, so let's remove field 4 if not 00
         local ip
         ip=$(flat_to_ipv6 "$(grep "$nic" /proc/net/if_inet6 | awk '$4==00 {print $1}')")
-        # IPv6 IPs should be surrounded by brackets to let ceph-monmap being happy
-        echo "[$ip]"
+        if [ -n "$ip" ]; then
+          # IPv6 IPs should be surrounded by brackets to let ceph-monmap being happy
+          echo "$ip"
```

Comment:
why are the brackets gone?

Reply:
Sorry for that. Mixed two different types of fixes together.